src/pages/__devOnly/DesignSystem.tsx

```diff
@@ -0,0 +1,68 @@
+import styled from 'styled-components'
+import { useFormik } from 'formik'
+
+import { Typography } from '~/components/designSystem'
+import { theme, PageHeader } from '~/styles'
+import { DatePickerField } from '~/components/form'
+
+const DesignSystem = () => {
+  const formikProps = useFormik({
+    initialValues: {
+      date: undefined,
+    },
+    onSubmit: () => {},
+  })
+
+  return (
+    <>
+      <PageHeader $withSide>
+        <Typography variant="bodyHl" color="textSecondary" noWrap>
+          Design System components
+        </Typography>
+        <Typography variant="caption">Only visible in dev mode</Typography>
+      </PageHeader>
+      <Container>
+        <Form onSubmit={(e) => e.preventDefault()}>
+          <Typography variant="headline">Form</Typography>
+          <Block>
+            <DatePickerField name="date" label="Date Picker" formikProps={formikProps} />
+            <DatePickerField
+              name="date"
+              label="Date Picker with helper"
```

Comment:
Shall we use translations instead ?

Reply:
Nop, it's dev only, I won't bother Mike for this, it's not even a regular page

Reply:
Oh yes!!! My bad